role_policy/hooks.py

```diff
@@ -26,6 +26,7 @@ def post_init_hook(cr, registry):
     users = env["res.users"].search([("share", "=", False)])
     users -= env.ref("base.user_admin")
     users -= env.ref("base.user_root")
+    users -= env.ref("base.public_user")
```

Comment:
this line is not needed since the public user is already filtered out via domain ("share", "=", False)